emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
+             "user_id" : ecwb.WrapperBase.Access.WORM,
+             "common_place_id" : ecwb.WrapperBase.Access.WORM
```

Comment:
Maybe my questions was not quite clear. I am asking why you need to construct a unique ID instead of using the Mongo generated id? Is there a particular reason for this?